internal/cluster/validations/validations.go

```diff
@@ -220,6 +221,20 @@ func ValidateBaseDNS(dnsDomainName, dnsDomainID, dnsProviderType string) error {
 	return validateBaseDNS(dnsDomainName, dnsDomainID, dnsProvider)
 }
 
+func ValidateNTPSource(ntpSource string) bool {
```

Comment:
Isn't it a validation that can be done by swagger?

Reply:
I think it is, yet I'm conflicted about using the wagger for this.
To my knowledge validations made by the swagger generated server are not logged properly and we might miss it.
This API is called by the agent, so it can just fail slightly until someone will look in the agent log.